to-do-app/src/AddTodo.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+
+class AddTodo extends Component {
+    state = {
+        content: ''
+    }
+    handleChange = (e) => {
+        this.setState({
+            content: e.target.value
+        })
+    }
+    handleSubmit = (e) => {
+        e.preventDefault();
+        this.props.addTodo(this.state);
+        this.setState({
+            content: '',
+            complete: false
+        })
+    }
+    render() {
+        return (
+            <div>
+                <form onSubmit={this.handleSubmit}>
+                    <label>What needs to bee done?</label>
```

Comment:
```suggestion
                    <label for="...">What needs to bee done?</label>
```
Without `for` or without wrapping input in label the label becomes useless

Reply:
Fixed